docs/resources/aws_security_group.md.erb

```diff
@@ -121,6 +163,30 @@ A String reflecting the name that was given to the SG at creation time.
       its('group_name') { should cmp 'my_group' }
     end
 
+### inbound\_rules
+
+A list of the rules that the security group applies to incoming network traffic. This is a low-level property that is used by the [`allow_in`](#allow_in) and [`open_to_the_world`](#open_to_the_world) matchers; see them for detailed examples. `inbound_rules` is provided here for those wishing to use Ruby code to inspect the rules directly, instead of using higher-level matchers.
+
+Order is very important in these rules, as the sequentially first rule to match will be applied to network traffic. By default, AWS includes a reject-all rule as the last inbound rule.  This implicit rule will not appear in the inbound_rules list.
+
+If the security group could not be found (that is, `exists` is false), `inbound_rules` will return an empty list.
+
+    describe aws_security_group(group_name: linux_servers) do
+      its('inbound_rules.first') { should include(from_port: '22', ip_ranges: ['10.2.17.0/24']) }
+    end
+
+### outbound\_rules
+
+A list of the rules that the security group applies to outgoing network traffic initiated by the AWS resource in the security group. This is a low-level property that is used by the [`allow_out`](#allow_out) matcher; see it for detailed examples. `outbound_rules` is provided here for those wishing to use Ruby code to inspect the rules directly, instead of using higher-levle matchers.
+
+Order is very important in these rules, as the sequentially first rule to match will be applied to network traffic. Outbound rules are typically used when it is desirable to restrict which portions of the internet, if any, a resource may access. By default, AWS includes an allow-all rule as the last outbound rule; note that Terraform removes this implicit rule. 
+
+If the security group could not be found (that is, `exists` is false), `outbound_rules` will return an empty list.
+
+    describe aws_security_group(group_name: isolated_servers) do
```

Comment:
Same notes as previous about tying the words above with the code sample below. `isolated_servers` is not defined.